docs/_embed/partner_api.md

```diff
@@ -348,6 +376,7 @@ Available parameters to the Zaps resource:
 | parameter                   | requirement | notes                                                                                                                                      |
 | --------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
 | **params\_\_{KEY}={VALUE}** | Optional    | Return Zaps that have a specific key/value set in the params (settings) of the Zap. |
+| **get_params** | Optional    | Return Zaps along with their associated key/value param settings. |
```

Comment:
I noticed this wasn't added to the changelog. Feel free to remove this for now, or make a note that this MR should not be merged as is until https://zapierorg.atlassian.net/browse/TPS-2398 is Done.

Reply:
The feature is merged, so all good here!

I would add an example of using it though - `/v1/zaps?get_params` will return a response where steps that belong to the partner's app have `"params": {"some_param": "some_value"}` while all other steps will have `"params": null"`.

Reply:
Updated the existing example to include the params field.